### PR TITLE
Update owners with JIRA

### PR DESCRIPTION
**Pull request by orsenthil**
## High-level description

This will be used with mergebot override feature.

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

https://jira.mesosphere.com/browse/DCOS-19647

### CODE REVIEW

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
@mesosphere-mergebot label ready for review

File: owners.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+  "adam-mesos": {"slack": "adam", "jira": "adam"},
+  "asridharan ": {"slack": "avinash", "jira": "avinash"},
+  "branden": {"slack": "branden", "jira": "branden"},
+  "gpaul": {"slack": "gpaul", "jira": "gpaul"},
+  "jgehrcke": {"slack": "jp", "jira": "jp"},
+  "klueska": {"slack": "klueska", "jira": "klueska"},
+  "lingmann": {"slack": "jeremy", "jira": "jeremy"},
+  "MatApple": {"slack": "mat", "jira": "mat"},
+  "mnaboka": {"slack": "mnaboka", "jira": "mnaboka"},
+  "mellenburg": {"slack": "mellenburg", "jira": "michael.ellenberg"},
```
Comment (mellenburg) [inline owners.json]: michael.ellenburg

**Comment by orsenthil** [comment]
@mellenburg  - addressed.

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
The tests were successful prior the spell fix. Applying  ship-it label.

**Comment by orsenthil** [comment]
@mesosphere-mergebot label ship-it

**Pull request merged**